### PR TITLE
Fixed `laravel-nginx-php-fpm` docker build fail. And added `laravel-bitnami`.

### LINKED ISSUE #35: docker build failed with Laravel
On my laptop
圖片

On github codespaces
圖片

### PR DESCRIPTION
**Pull request by dwchiang**
- Modified: service `laravel` to `laravel-nginx-php-fpm`, running on  port 9095.
- Fixed: docker build fail on `laravel-nginx-php-fpm`
- Added: service `laravel-bitnami`, running on port 9096.

Fixes #35

### CODE REVIEW

File: services/laravel-bitnami/laravel-8.5.20/database/factories/UserFactory.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->name(),
+            'email' => $this->faker->unique()->safeEmail(),
+            'email_verified_at' => now(),
+            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
```
Comment (pahud) [inline services/laravel-bitnami/laravel-8.5.20/database/factories/UserFactory.php:29]: Is it safe to hardcode the string here?

**Review by pahud — COMMENTED** [review]
LGTM @neilkuan  can you take a look as well?

File: services/laravel-bitnami/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+!logs
```
Comment (neilkuan) [inline services/laravel-bitnami/.gitignore:1]: Does this file still need to exist?

Comment (pahud) [reply services/laravel-bitnami/.gitignore:1]: @dwchiang

Comment (dwchiang) [reply services/laravel-bitnami/.gitignore:1]: Let me remove them :)

Comment (dwchiang) [reply services/laravel-bitnami/.gitignore:1]: Oh, I think this `.gitignore` need to be here to unignore the `storage/logs` folder which is required for Laravel framework.

I will remove the unused files under `./php` folder.

**Review by pahud — APPROVED** [review]

**Pull request merged**